pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -191,8 +191,10 @@ installPhase() {
                     --set-rpath $out/lib:$libPath $bin/bin/$i
             fi
         done
-        # FIXME: needs PATH and other fixes
-        # install -Dm755 nvidia-bug-report.sh $bin/bin/nvidia-bug-report.sh
+        substituteInPlace nvidia-bug-report.sh \
+          --replace /bin/grep grep \
+          --replace /bin/ls ls
+        install -Dm755 nvidia-bug-report.sh $bin/bin/nvidia-bug-report.sh
```

Comment:
could we maybe add a call to `wrapProgram` here to ensure these (and other tools) are actually available at runtime?